File: Java/HelloWorld.java
import java.lang.management.ManagementFactory;

public class HelloWorld {
    
    public static void main(String[] args) {

        long startTime = System.currentTimeMillis(); // start the timer

        for (int i = 1; i <= 10000; i++) { // for loop to print "Hello World!" 10000 times
            System.out.println("Hello World!");
        }

        // get the current memory usage

        Runtime runtime = Runtime.getRuntime(); 
        runtime.gc();
        
        long memory = runtime.totalMemory() - runtime.freeMemory();
        System.out.println("Used memory is : " + memory + " bytes");

        // end the timer

        long stopTime = System.currentTimeMillis(); 
        long elapsedTime = stopTime - startTime;
        System.out.println("Elapsed Time : " + elapsedTime + " ms");

        // get the current cpu usage

        com.sun.management.OperatingSystemMXBean osBean = ManagementFactory.getPlatformMXBean(com.sun.management.OperatingSystemMXBean.class);
        System.out.println(osBean.getProcessCpuLoad() * 100);
        System.out.println("System Load Average : " + osBean.getProcessCpuLoad() * 100);
        
    }
}

File: Python/HelloWorld.py
import os
import psutil
import time


def my_func(): 

  # start the timer

  start_time = time.time()

  # for loop to print 10000 "Hello World!"

  for line in range(10000):
    print("Hello World!")

  # get and print the runtime

  print("--- %s seconds ---" % (time.time() - start_time))
  
  # get and print the current cpu usage

  print("CPU : %s" % psutil.cpu_percent())
  
  # get the current memory usage

  process = psutil.Process(os.getpid())
  
  # print the current memory usage

  print("Memory : %s bytes" % process.memory_info().rss)

if __name__ == '__main__':
    my_func()